app/lib/common/widget/InviteListView.dart

```diff
@@ -0,0 +1,84 @@
+// ignore_for_file: prefer_const_constructors, require_trailing_commas
+
+import 'package:effektio/common/store/separatedThemes.dart';
+import 'package:flutter/material.dart';
+
+class InviteListView extends StatefulWidget {
+  const InviteListView({
+    Key? key,
+    required this.name,
+    required this.isAdded,
+  }) : super(key: key);
+
+  final String name;
+  final bool isAdded;
+
+  @override
+  InviteListViewState createState() => InviteListViewState();
+}
+
+class InviteListViewState extends State<InviteListView> {
+  static const TextStyle optionStyle =
```

Comment:
Can you put this in seperatedThemes? It is a theme component.

Reply:
Noted